### ISSUE TITLE
No red asterisk, should there be language saying one of the following is required?

### ISSUE THREAD

**Issue by CWEmrich** [LemmaLegalConsulting/docassemble-StudentEvaluations#41]
https://github.com/LemmaLegalConsulting/docassemble-StudentEvaluations/blob/9124749346a007273b266ff5f0498ae969774ecd/docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml#L251-L262

**Comment by miabonardi** [comment]
@nonprofittechy What do you think

**Comment by nonprofittechy** [comment]
Maybe rephrase: how can the school reach you? you need to tell us at least one way to get in contact with you.

**Comment by miabonardi** [comment]
@CWEmrich
